lib/ts/recipe/emailpassword/types.ts

```diff
@@ -116,9 +116,13 @@ export type RecipeInterface = {
     getUserByEmail(input: { email: string; userContext: any }): Promise<User | undefined>;
 
     createResetPasswordToken(input: {
-        userId: string;
+        userId: string; // the id can be either recipeUserId or primaryUserId
         userContext: any;
-    }): Promise<{ status: "OK"; token: string } | { status: "UNKNOWN_USER_ID_ERROR" }>;
+    }): Promise<
+        | { status: "OK"; token: string }
+        | { status: "UNKNOWN_USER_ID_ERROR" }
+        | { status: "PROVIDE_RECIPE_USER_ID_AS_USER_ID_ERROR" }
```

Comment:
PROVIDE_RECIPE_USER_ID_AS_USER_ID_ERROR -> makes no sense to me. Whats this for?